libraries/AP_Math/transform.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include "AP_Math.h"
+
+// A homogonous transform type that matches ROS 2 geometry_msg/msg/TransformStamped
+struct TransformF {
```

Comment:
Perhaps we should have this as a separate cleanup later, and for now use the existing entry-points into AP_VisualOdom?

Reply:
Sure! I kicked it off to `add-transform-header` which can go in later.